src/Microsoft.VisualStudio.Editors/SettingsDesigner/TypeEditorDataGridViewColumn.vb

```diff
@@ -25,5 +28,23 @@ Namespace Microsoft.VisualStudio.Editors.SettingsDesigner
                 MyBase.CellTemplate = Value
             End Set
         End Property
+
+        Protected Function OnCreateAutomationPeer() As AutomationPeer
+            Return New DataGridViewUITypeEditorColumnAutomationPeer(_userControl)
```

Comment:
`_userControl` is never initialised, so will always be `null`. Looking at the base classes (`UserControlAutomationPeer`, which derives from `FrameworkElementAutomationPeer`, which derives from `UIElementAutomationPeer`) I see it supposedly [throws an exception](https://referencesource.microsoft.com/#PresentationCore/Core/CSharp/System/Windows/Automation/Peers/UIElementAutomationPeer.cs,27) if the `owner` argument is null, so I'm surprised this works. If you run this in a debugger and break on all exceptions, do you see an `ArgumentNullException` here?

`DataGridViewColumn` is a WinForms type. `AutomationPeer` is a WPF type. Again, I'm surprised this works!

Reply:
VS was playing with me 😕 I'm testing this again and I don't get the same results, but I'm sure it has to do with this. Let me investigate a little bit more.